src/object/srv_mod.c

```diff
@@ -106,7 +106,7 @@ obj_tls_fini(const struct dss_thread_local_storage *dtls,
 		migrate_pool_tls_destroy(pool_tls);
 
 	if (tls->ot_echo_sgl.sg_iovs != NULL)
-		daos_sgl_fini(&tls->ot_echo_sgl, true);
+		d_sgl_fini(&tls->ot_echo_sgl, true);
```

Comment:
You don't technically need the conditional here although it will save a function call.  There seem to be a number of places like this, in theory you could probably make d_sgl_free() inline or a macro and it would probably save a few cycles but outside the scope of this PR.

Reply:
Agree.